api/concept/type/AttributeType.java

```diff
@@ -38,9 +38,6 @@ default ValueType getValueType() {
         return ValueType.OBJECT;
     }
 
-    @CheckReturnValue
-    boolean isKeyable();
```

Comment:
This was unused and just calling `getValueType().isKeyable()`, so it was unnecessary.

Reply:
good catch!